src/data/finset/lattice.lean

```diff
@@ -47,6 +47,18 @@ fold_image_idem
   (s.map f).sup g = s.sup (g ∘ f) :=
 fold_map
 
+lemma sup_map_multiset {α : Type*} [decidable_eq α] [decidable_eq β]
```

Comment:
```suggestion
lemma multiset_map_sup {α : Type*} [decidable_eq α] [decidable_eq β]
```

Reply:
I think this should be `multiset.map_finset_sup` to match `multiset.count_finset_sup`